LayoutTests/svg/animations/animate-update-crash.xhtml

```diff
@@ -1,20 +0,0 @@
-<html xmlns="http://www.w3.org/1999/xhtml">
-    <body>
-        <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-            <text>
-                PASS
-                <textPath>
-                    <tref xlink:href="#foo">
-                        <animateColor attributeName="keyPoints"></animateColor>
-                        <animateColor attributeName="xlink:href"></animateColor>
-                    </tref>

```

Comment:
This is likely still a valid crash test, since tref does not seem to be the focus here but animation is. It is likely enough to substitute tref for another svg tag here (maybe <a> if it needs to have explicit xlink:href support).